include/pstl_offload/algorithm

```diff
@@ -0,0 +1,28 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
```

Comment:
The new files are not a part of LLVM project, so this portion should be removed from the copyright headers.

Reply:
I have updated all of the copyrights in new files with the year 2023 and removed LLVM part.